src/common/utils/docker.ts

```diff
@@ -54,6 +54,11 @@ export const pushImage = async (image_ref: string) => {
   await docker(['push', image_ref]);
 };
 
+export const getDigest = async (image_ref: string): Promise<string> => {
+  const digest = await docker([`inspect`, `--format='{{index .RepoDigests 0}}'`, image_ref], { stdout: false });
+  return digest.stdout.split('@')[1].replace('\'', '');
```

Comment:
Do you want to replace every instance or only the first one?

Reply:
There is only one instance. Since it's `stdout` it's getting `'repository@sha256:abcd'` and I just want to get rid of the trailing apostrophe after the split.